torch/testing/_internal/common_distributed.py

```diff
@@ -842,6 +854,66 @@ def _check_return_codes(self, elapsed_time) -> None:
     def is_master(self) -> bool:
         return self.rank == 0
 
+# Utility base class for DDP based Multi Process Test cases
+# This abstracts the PG creation and deletion, the backends are selected based
+# on device type. The tests functions can be instantiated per device type using
+# common_device_type.instantiate_device_type_tests
+# other backends can add entry in backend() function
+class DistributedTestBase(MultiProcessTestCase):
```

Comment:
Is it used anywhere?

Reply:
This will the new Base class to be used in the test/distributed files which abstracts out the device.